src/apps/service/apps.clj

```diff
@@ -210,16 +210,28 @@
     (format-job-submission-response job-info)))
 
 (defn update-job-status
-  ([external-id status end-date]
-     (let [{job-id :job_id} (jobs/get-unique-job-step external-id)]
-       (update-job-status job-id external-id status end-date)))
+  ([external-id]
```

Comment:
Just looking at this now, but should this whole block be in a transaction? (or, will it be already from somewhere else that I missed, heh)

Reply:
D'oh! Yes it should. Thanks for catching that.